### PR TITLE
use branch newzealidar v0.1

### PR DESCRIPTION
**Pull request by xandercai**
DESCRIPTION OF PR:

## Developer Checklist
- [x] Make code change
- [ ] Update tests
  - [ ] Update / create new tests
  - [ ] Ensure these tests have the expected behaviour
  - [ ] Test locally and ensure tests are passing
- [ ] Update documentation
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

## Reviewer Checklist
- [ ] Check new code for code smells
- [ ] Check new tests
  - [ ] Ensure adequate coverage
  - [ ] Check for code smells within tests
- [ ] Check if documentation needs updating
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

### CODE REVIEW

**Review by LukeParky — APPROVED** [review]
Fixing the version is a good idea. Approve

File: environment.yml [hunk 1]
```diff
@@ -31,6 +31,6 @@ dependencies:
     - geovoronoi==0.4.0
     - celery==5.2.7
     - gunicorn==20.1.0
-    - git+https://github.com/xandercai/NewZeaLiDAR
+    - git+https://github.com/xandercai/NewZeaLiDAR/tree/v0.1
```
Comment (caseyli92) [inline environment.yml:35]: I'm currently having a problem creating the environment with this new link here.

Comment (xandercai) [reply environment.yml]: It turns out that pip cannot install a git branch by ".../tree/branch" way; need to use "....git/@branch" way.
I have committed an update to the PR. Sorry about the mistake.

see [ref](https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch)

**Review by caseyli92 — APPROVED** [review]
Tried this, and it works now :)

**Pull request merged**